modules/social_features/social_follow_taxonomy/src/Plugin/views/filter/FollowTaxonomyViewsFilter.php

```diff
@@ -0,0 +1,288 @@
+<?php
+
+namespace Drupal\social_follow_taxonomy\Plugin\views\filter;
+
+use Drupal\Core\Entity\Element\EntityAutocomplete;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\flag\FlagInterface;
+use Drupal\taxonomy\Plugin\views\filter\TaxonomyIndexTid;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Filters users by flagged taxonomies in a view.
+ *
+ * @ViewsFilter("social_follow_taxonomy_follow_filter")
+ */
+class FollowTaxonomyViewsFilter extends TaxonomyIndexTid {
+
+  /**
+   * The target flag entity id.
+   */
+  const FLAG_ID = 'follow_term';
+
+  /**
+   * The entity manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
```

Comment:
It's not an issue, but now we support min PHP 7.4 and we can write like:
```
  /**
   * The entity manager.
   */
  protected EntityTypeManagerInterface $entityTypeManager;
```

Reply:
Fixed.